hassio/src/system/hassio-system-metrics.ts

```diff
@@ -0,0 +1,189 @@
+import "@material/mwc-button";
+import "@material/mwc-list/mwc-list-item";
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  internalProperty,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+import { classMap } from "lit-html/directives/class-map";
+import memoizeOne from "memoize-one";
+import "../../../src/components/buttons/ha-progress-button";
+import "../../../src/components/ha-bar";
+import "../../../src/components/ha-button-menu";
+import "../../../src/components/ha-card";
+import "../../../src/components/ha-settings-row";
+import { fetchHassioStats } from "../../../src/data/hassio/common";
+import { HassioHostInfo } from "../../../src/data/hassio/host";
+import { haStyle } from "../../../src/resources/styles";
+import { HomeAssistant } from "../../../src/types";
+import {
+  getValueInPercentage,
+  roundWithOneDecimal,
+} from "../../../src/util/calculate";
+import { hassioStyle } from "../resources/hassio-style";
+
+const TARGET = 85;
```

Comment:
Can we make it a 3 stage? Normal/warning/critical

Reply:
Sure we can!

< 50 green
50 - 85 amber
> 85 red
?

I'm also thinking I should make an "hassio-metrics" component, this was more duplicate code then what I imagined.

Reply:
Could just create a render function that takes a name and value as parameter? Or will we use this for addons too?

Reply:
The idea is eventually addons as well, but it might need a different approach, a simple renderer will do for now :+1: